python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -189,6 +196,12 @@ def setDefaults(self):
         # after the user has set doPreConvolve.
         self.measurement.algorithms.names.add('base_PeakLikelihoodFlux')
 
+        self.forcedMeasurement.plugins = ["base_TransformedCentroid", "base_PsfFlux"]
```

Comment:
Just to verify these are the only plugins you want right? I assume it is doing what you want since you must have tested it, but sometimes people get bitten by =, |=, +=, etc for this datatype so I wanted to double check.